### ISSUE TITLE
Some functions are not working anymore

### ISSUE THREAD

**Issue by godhand2** [recnaleerf/Ancheera#30]
Title. This happened yesterday evening and I'm not sure what is the cause of the problem. 
Now the AP/EP tracker, daily quest counter, and the supply tab aren't working anymore. Tried to re-install both chrome and ancheera but both ended up not fixing it.
@daitan Would you please fix this?

**Comment by dragonz7** [comment]
Adding to this: It seems like the new update broke a lot of things. Auto end quest and skip quest next doesn't work anymore.

**Comment by Nephilimus** [comment]
Yup, +1 to the trackers no longer working, including the pendant trackers. Ancheera has become indispensable, so I hope these issues will be fixed soon. :(

**Comment by daitan** [comment]
I'm afraid to say that I can't find a clean work around for this current issue that [chrome 65](https://bugs.chromium.org/p/chromium/issues/detail?id=801306) has introduced. Not without a major rewrite at the very least.

Your best bets currently are to roll back your chrome version or use a chromium version which does not have these issues present if you still need to run Ancheera.

**Comment by godhand2** [comment]
@daitan 
Rolled back to chrome 63 and ancheera is working fine, thank you for your time. Also, if the bug you mentioned get fixed (let's say chrome 66), will ancheera be able to run w/o being mostly broken on that version?

**Comment by daitan** [comment]
I’m assuming it will be fixed in the next stable release as there is a bugtracker for it. 

If it’s still broken I’ll look into alternatives.

**Comment by godhand2** [comment]
Thank you for letting me know, much appreciated.

**Comment by daitan** [comment]
Recent chrome update to version 66.0.3359.117 appears to have fixed their issue on their end, let me know if problems persist with ancheera.
